Week1/homework/js-exercises/6-log-an-array.js

```diff
@@ -0,0 +1,16 @@
+// Exercise 6: Log an array of animals
+
+{
+    'use stict';
+
+    let items = [];
+    console.log("the value of an array is : []");
+    console.log(items);
+
+    let favAnimals = ["cat","dog","horse"];
+    console.log(favAnimals);
+    favAnimals.unshift("lion");
+    console.log(favAnimals);
+
```

Comment:
`const` here instead of `let` please

Reply:
but we are manipulating the content of an array here, 
should I still use the `const` here ?